irods_browser_app/static/css/site_base_irods.css

```diff
@@ -21,30 +17,13 @@
     color: #357ebd;
 }
 
-#irodsContent .modal-body {
-    height: 342px;
-}
-
-#irodsContent #irods_structure {
-    width: 517px;
-    height: 300px;
-    padding: 10px;
-    float: left;
-    box-shadow: 1px 1px 3px 1px #000;
-}
-
-#irodsContent #irods_structure #viewing input {
-    padding-left: 10px;
-    padding-right: 10px;
-    border-radius: 10px;
-    margin-left: 5px;
-    width: 420px;
-}
-
```

Comment:
@mauriel133 I believe these classes are also still used by irods browser html files. Same comments as above. Just need to double check to either keep them or make sure they are replaced with better style definitions.